### DISCUSSION TITLE
Volume up down / no modifier / config keybind

### DISCUSSION THREAD

**Discussion by NicTanghe** [leftwm/leftwm#912] [Q&A] [answered]
Hello I`d like to add the following shortcuts to my config .ron 

```rust
(command: Execute, value: "amixer set Master 10%+", modifier: [], key: "VolumeUp"),
(command: Execute, value: "amixer set Master 10%-", modifier: [], key: "VolumeDown"),
```

It doesn`t seem to work though.

**Comment by VuiMuich** [comment, marked as answer]
You can use `xev -event keyboard` to find the exact name of your volume keys, then compare the output of the above command with [this list](https://github.com/leftwm/lefthk/blob/main/lefthk-core/src/xkeysym_lookup.rs) to check if the xlib uses a slightly different name.

**Reply by Eskaan** [reply]
for me it is
```rust
        (command: Execute, value: "amixer -q set Master 5%+", modifier: [], key: "XF86XK_AudioRaiseVolume"),
        (command: Execute, value: "amixer -q set Master 5%-", modifier: [], key: "XF86XK_AudioLowerVolume"),
```

Also the list VuiMuich forgot to link:
https://github.com/leftwm/lefthk/blob/main/lefthk-core/src/xkeysym_lookup.rs

**Reply by NicTanghe** [reply]
THX,
 now to find a way to update the eww bar when scrolling.
